docs/config_objects.rst

```diff
@@ -134,9 +134,13 @@ Galaxy Types
 
     * ``index`` = *int_value* (default = 'Sequence' from 0 to ``cosmos_catalog.nobjects``-1)  Which item in the catalog to use. Special: If ``index`` is either a 'Sequence' or 'Random' and ``last`` or ``max`` (respectively) is not specified, then it is automatically set to ``real_catalog.nobjects-1``.
     * ``gal_type`` = *str_vale* (required, unless ``real_catalog.use_real`` is False, in which case 'parametric') Which type of galaxy to use.  Options are 'real' or 'parametric'.
+    * ``noise_pad_size`` = *float_val* (default = 5) The size of a padding region around the HST image when ``gal_type='real'``.
```

Comment:
I think this could use a little more explanation - what does the value mean?  e.g., does the default 5 mean that the postage stamp is enlarged by 5 pixels in each direction, or a factor of 5, or ...?

Reply:
Good point.  Will add.  I had to look it up -- It's the minimum size in arcsec to pad out the noise for the real galaxy stamps.  I guess many of them must be smaller than 5 arcsec, so this at least sets that as a uniform minimum size of the noise region.